kytos/core/connection.py

```diff
@@ -90,8 +99,8 @@ def send(self, buffer):
         """
         try:
             if self.is_alive():
-                self.socket.sendall(buffer)
-        except (OSError, SocketError) as exception:
```

Comment:
Deleted SocketError exception since we are using a transport not a socket. I think it would be better to have only `Exception` here. Any raised exception from here would be silent and not shown in kytos console.

Reply:
On this `try catch` it's trying to gracefully close the socket/transport, but notice that it's also reraising with `raise`, which will be handled by handler msg out [here](https://github.com/kytos-ng/kytos/blob/master/kytos/core/controller.py#L629-L631), and it'll try to publish a conn error. Having a broad exception wouldn't be ideal, we should only consider including it if we can't narrow down which excs will be thrown, otherwise it'd make `msg_out_event_handler` much more complicated and broad to handle. That said, I'll recommend that you do some research looking into upstream source code to see if there's any other exc that can be raised.

Reply:
Added `TypeError`. `OSError` looks like it is a superclass of the deprecrated `socket.error`, [info],(https://docs.python.org/3.11/library/socket.html#socket.error) and catches transport errors, [documentation](https://docs.python.org/3/library/errno.html#errno.EISCONN).

Reply:
Removed `TypeError`.